src/receive_imf.rs

```diff
@@ -2334,10 +2339,12 @@ struct GroupChangesInfo {
     /// Optional: A better message that should replace the original system message.
     /// If this is an empty string, the original system message should be trashed.
     better_msg: Option<String>,
+    /// Added/removed contact `better_msg` refers to,
```

Comment:
```suggestion
    /// Added/removed contact `better_msg` refers to
```

or did you mean to add something behind the `,`?

Reply:
no, but i changed it to `.` as for the other comments